### ISSUE TITLE
Made it possible to wrap commands in if, elif, and else conditions.

### ISSUE THREAD

**Issue by lovelle-cardoso** [ImpowerGames/impower#150]
To test:

```
var test = true
var x = 5

* if (test):
  * if (x < 5):
    Too low...
  * elif (x == 5):
    Perfect!
    So...
    Now what?
  * else:
    Too high!!
    
ALL DONE!
```

**Comment by Wipeless** [comment]
@lovelle-cardoso  When setting `x` to something greater than 5, I am unable to reach the else's "Too high!!"

It goes straight to "ALL DONE!"

**Comment by lovelle-cardoso** [comment]
@Wipeless Fixed by 05cfe8ca

**Comment by LemBurnstein** [comment]
I was able to get the Too high result
